.changeset/forty-beers-sniff.md

```diff
@@ -0,0 +1,38 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-catalog-graph': patch
+---
+
+Added possibility to show arrow heads for graph edges for better understandability.
+
+In order to show arrow heads in the catalog graph page, add `showArrowHeads` attribute to `CatalogGraphPage` component
+(typically in `packages/app/src/App.tsx`):
+
+```diff
+  <CatalogGraphPage
```

Comment:
Hmm, I took this from the App.tsx in the dev app, but I see now that it would be better to use the one in the create-app template which is much simpler out of the box, one more iteration.

Reply:
Yeah, super sorry, all I was looking for was a simple one-liner to show the new prop as a hint. If we can simplify this that would be great. Again, sorry, really not trying to make this a painful process :(

Reply:
Simplified

Reply:
No worries, thanks for fast feedback, makes it much less painful.